resources/js/app.js

```diff
@@ -16,6 +16,33 @@ require('./bootstrap');
 $('select.selectize').selectize({ maxItems: 3 });
 $('textarea.wysiwyg').markdown({ iconlibrary: 'fa' });
 
+$('.dropdown').on('click', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    const activate = $(this).next().hasClass('active') ? false : true;
+    $('.subnav').removeClass('active');
+    if(activate) {
```

Comment:
Also add some white lines here to let the code breath a little.

```suggestion
    if (activate) {
```